manifests/06-trusted-ca-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+# The network operator is responsible for injecting
+# the trusted ca bundle into this configmap.
```

Comment:
Right. As we discussed in Slack. The two configmaps are from different namespaces as since they are consumed as volume mounts we cannot consume the operator's one in the operand's namespace.